src/functions.js

```diff
@@ -0,0 +1,36 @@
+function addTask(tasks, input) {
+  if (input.value.trim() === '') return;
+
+  tasks.push({
+    description: input.value,
+    completed: false,
+    index: tasks.length,
+  });
+
+  input.value = '';
+
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+function updateTask(tasks, task) {
+  tasks[task.index - 1] = task;
+
+  document
+    .querySelectorAll('li')
+    .forEach((elm) => elm.classList.remove('active'));
+
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+function deleteTask(tasks, index) {
+  tasks = tasks.filter((t) => t.index !== index);
```

Comment:
[OPTIONAL]

- I would highly recommend using proper ES6 syntax instead of the `function` keyword. As it is very useful to write clean code and also it achieve the goal in less code.